app/ui/placeholder/src/main/kotlin/com/hedvig/android/placeholder/PlaceholderHightlightMaterial3.kt

```diff
@@ -39,7 +40,10 @@ fun PlaceholderHighlight.Companion.shimmer(
   animationSpec: InfiniteRepeatableSpec<Float> = PlaceholderDefaults.shimmerAnimationSpec,
   @FloatRange(from = 0.0, to = 1.0) progressForMaxAlpha: Float = 0.6f,
 ): PlaceholderHighlight = PlaceholderHighlight.shimmer(
-  highlightColor = PlaceholderDefaults.shimmerHighlightColor(),
+  highlightColor = PlaceholderDefaults.shimmerHighlightColor(
+    backgroundColor = MaterialTheme.colorScheme.surface,
+    alpha = 1f,
```

Comment:
How did the default of 0.75f look like here? did you feel like it was way too subtle?

Reply:
a bit, but I've put it back to default now:) it's not that much of difference